### ISSUE TITLE
Help Qgis not opening any project on my PC after crush while loading a WFS layer

### ISSUE THREAD

**Issue by mb09100** [qgis/QGIS#58977]
### What is the bug or the crash?

I was trying to upload a layer froma emodnet service and the Qgis went non responsive. After ending task though task manage. I have tried to reopen the project but it never goes past 87% load and it stays there infinite with also toher projects I try to open. I am desperate as I am presently working on a project to deliver tomorrow and I must find a way to use QGIS on my pc

### Steps to reproduce the issue

Load Layer > WFS Layer > new > https://ows.emodnet-seabedhabitats.eu/geoserver/emodnet_open/wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0 > I tried to load the Cyprus seabed habitat  map

### Versions

3.28

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by rouault** [comment]
Doesn' seem like a QGIS bug, but rather than the server is dead / non-responsive.  "https://ows.emodnet-seabedhabitats.eu/geoserver/emodnet_open/wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0" stalls forever

**Comment by mb09100** [comment]
Yes but that messed up everything. Now I can't open any project in my qgis. Apart from a blank one. There must be some setting in my pc. How can I delete a virtual layer from database? What If I wanted delete that last connection to that layer in question from db

**Comment by rduivenvoorde** [comment]
Maybe you can start a new profile first? Can you start QGIS from a command line? 
Something like "qgis --profile foo".

Then via menu: Settings/User Profiles/Open active profile folder you can go to your 'Default profile folder' (given you did not work from another one, it would be in the parent folder).

There you will find a file QGIS/QGIS.ini

It is a text file which you can edit.

In there are all connections you have in that profile. 

Search for `ows\items\wfs\connections` and remove all lines from the connection that stalls your QGIS.

It could be that you have a browser window open? Not sure but with the browserwindow open, it tried to do getcapability requests to your connections and maybe these are failing?
